tox.ini

```diff
@@ -18,41 +23,57 @@ commands = pre-commit run --all-files --show-diff-on-failure --color=always
 [testenv:py{,38,39,310,311,312}{,-fast}]
 depends: pre-commit
 use_develop = fast: True
-skip_sdist = fast: True
```

Comment:
Originally, I thought that `skip_sdist` would help because Briefcase was being installed in editable mode....but `tox` actually builds an "editable package" and installs that. So, `skip_sdist` wasn't doing anything....but building and installing the "editable package" is much faster.